Add --enable-insecure-extension-access to install extension with auto

<!--  PLEASE FILL THIS OUT, IT WILL MAKE BOTH OF OUR LIVES EASIER -->

**Has this issue been opened before?**

- [ x] It is not in the [FAQ](https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/FAQ), I checked.
- [ x] It is not in the [issues](https://github.com/AbdBarho/stable-diffusion-webui-docker/issues?q=), I searched.


**Describe the bug**

When trying to install an extension i have this :
[Bug]: AssertionError: extension access disabed because of commandline flags


**Which UI**

auto


**Hardware / Software**
 - OS: windowds 11
 - OS version: 22H2 (OS Build 22621.819)
 - WSL version (if applicable): docker-desktop V2d
 - Docker Version: Docker Desktop 4.13.0 (89412)
 - Docker compose version:v2.12.0
 - Repo version: release 3.2.0
 - RAM: 32
 - GPU/VRAM: 3090 24giga

**Steps to Reproduce**
1. Go to extensions
2. Go to availables
3. Try to install dreambooth
4. See error

**Additional context**
By adding --enable-insecure-extension-access to the CLI_ARGS it work, it's linked to this change :
https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/ccf1a15412ef6b518f9f54cc26a0ee5edf458108#diff-15faa364afc7fd40ccc1e6a38893b2dac62c846715710dfbe44f25fb65205440

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -29,7 +29,7 @@ services:
     build: ./services/AUTOMATIC1111
     image: sd-auto:18
     environment:
-      - CLI_ARGS=--allow-code --medvram --xformers
+      - CLI_ARGS=--allow-code --medvram --xformers --enable-insecure-extension-access
 
   auto-cpu:
     <<: *automatic

